Makefile

```diff
@@ -34,7 +34,7 @@ ifdef ENABLE_BLOBFUSE_PROXY
 override E2E_HELM_OPTIONS := $(E2E_HELM_OPTIONS) --set controller.logLevel=6 --set node.logLevel=6 --set node.enableBlobfuseProxy=true
 endif
 E2E_HELM_OPTIONS += ${EXTRA_HELM_OPTIONS}
-GINKGO_FLAGS = -ginkgo.v
+GINKGO_FLAGS = -ginkgo.v -ginkgo.timeout=24h
```

Comment:
what's the original value? 24h is too long

Reply:
original value is 1 hour, some test case running longer than that

Reply:
then set as 2h

Reply:
ok